src/Renderer/TrafficRenderer.cpp

```diff
@@ -59,14 +61,13 @@ TrafficRenderer::Draw(Canvas &canvas, const TrafficLook &traffic_look,
       canvas.Select(traffic_look.alarm_brush);
       break;
     case FlarmTraffic::AlarmType::NONE:
-      if(colorful_traffic){} // avoid unused variable warning
-      if (traffic.relative_altitude > (const RoughAltitude)50) {
-        canvas.Select(traffic_look.safe_above_brush);
-      } else if (traffic.relative_altitude > (const RoughAltitude)-50) {
-        canvas.Select(traffic_look.warning_in_altitude_range_brush);
-      } else {
-        canvas.Select(traffic_look.safe_below_brush);
-      }
+        if(colorful_traffic)
+        {
+          canvas.Select(traffic_look.colorful_traffic_brushes[indicators.rel_alt][indicators.climb]);
+        }
+        else {
+          canvas.Select(traffic_look.basic_traffic_brushes[indicators.rel_alt]);
```

Comment:
Something must be wrong here. `basic_traffic_brushes` has only 2 items, but `colorful_traffic_brushes` has 3 items. Both use the same index. Either one index is never used or this is a buffer overflow bug.

Reply:
bug... will fix